### PR TITLE
doc/rados: rewrite mclock docs (2 of 2)

### PR DESCRIPTION
**Pull request by zdover23**
## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by neha-ojha** [comment]
@zdover23 this needs a rebase now

File: doc/rados/configuration/mclock-config-ref.rst [hunk 1]
```diff
@@ -4,21 +4,20 @@
 
 .. index:: mclock; configuration
 
-Mclock profiles mask the low level details from users, making it
-easier for them to configure mclock.  
+Dmclock-related configuration parameters can be applied by using mclock
```
Comment (neha-ojha) [inline doc/rados/configuration/mclock-config-ref.rst:7]: Didn't we finalize this section in your earlier PR https://github.com/ceph/ceph/pull/40570/commits/e262e85606c9e4090454e902b5102954b8020313#diff-ec476ef02ccd9d5bdec6c63b3b4fb2eb6f001090a9fd2f6a59ad9d40403cb548R5-R21?

File: doc/rados/configuration/mclock-config-ref.rst [hunk 2]
```diff
@@ -174,74 +172,79 @@ maximize the impact of the mclock scheduler.
...
+configuration file called **[osd.x]**.
 
 Alternatively, the following commands may be used,
 
-``$ ceph config set [global, osd] osd_mclock_max_capacity_iops_[hdd,ssd]
-<value>``
+.. prompt:: bash $
+
+  ceph config set [global, osd] osd_mclock_max_capacity_iops_[hdd,ssd]
+  <value>
```
Comment (sseshasa) [inline doc/rados/configuration/mclock-config-ref.rst:229]: <value> should be a part of the previous line:

ceph config set [global, osd] osd_mclock_max_capacity_iops_[hdd,ssd] <value>

**Review by sseshasa — CHANGES_REQUESTED** [review]
Apart from the rebase required from the previous review, it looks good.

File: doc/rados/configuration/mclock-config-ref.rst (hunk 1 above)
Comment (zdover23) [reply doc/rados/configuration/mclock-config-ref.rst:7]: We did.

I've attempted to solve this problem by running these commands:

git checkout wip-doc-mclock-rewrite-second-half-2021-apr-2
git merge --patch master doc/rados/configuration/mclock-config-ref.rst

File: doc/rados/configuration/mclock-config-ref.rst (hunk 2 above)
Comment (zdover23) [reply doc/rados/configuration/mclock-config-ref.rst:229]: Done, but I'm not sure if it makes a difference when the docs are rendered.

**Review by sseshasa — COMMENTED** [review]
@zdover23 With the latest push all your previous changes are lost. You would have to re-apply your previous changes again.

**Comment by zdover23** [comment]
> @zdover23 With the latest push all your previous changes are lost. You would have to re-apply your previous changes again.

Aw man.

Stand by while I fix this up.

**Comment by zdover23** [comment]
Well, I've royally messed this one up. Everyone avert your eyes while I figure out how I want to deal with this.

**Comment by zdover23** [comment]
I'm closing this PR, and opening https://github.com/ceph/ceph/pull/40723 to address what I meant for this PR to address.

**Pull request closed without merging**